### DISCUSSION TITLE
Is it possible to set up fail2ban for shadowsocks?

### DISCUSSION THREAD

**Discussion by Steve-Brule** [qdm12/gluetun#889] [Q&A]
Hello. Love your work and the container! I was wondering if there's a way to enable fail2ban for shadowsocks authentication, as I want to expose the shadowsocks server to the internet. Is this possible?

**Comment by qdm12** [comment]
I have no idea, you could perhaps try `docker exec -it gluetun /bin/sh` and try to setup fail2ban there?

On the other hand, bad authentication requests won't really hurt the shadowsocks server, I believe it would just drop the packet and not respond, behaving like the packet is dropped like fail2ban does. I may be wrong though, I haven't checked in details.
